src/content/advanced/gitops/index.md

```diff
@@ -125,367 +123,193 @@ flux-multi-tenancy:
 
 Due to extra security policies enforced by Kyverno, setting `.spec.serviceAccountName` for `Kustomization`/`HelmRelease` resources in our Management Clusters is mandatory. Usually, you will want to use `serviceAccountName: "automation"`.
 
-### Setting up sources
+## Creating your repo structure
 
-First thing's first - create a bare-bones `GitRepository` resource that points to [giantswarm/flux-demo](https://github.com/giantswarm/flux-demo).
+A gitops repository can have any arbitrary structure. This provides flexibility to adapt the repo to your own infra sructure. In giant swarm we have a clear hierarchical structure and we recommend to use this structure. This provides some advantages:
 
-```nohighlight
-flux create source git flux-demo \
-        --url=https://github.com/giantswarm/flux-demo \
-        --branch=main \
-        --interval=30s \
-        --namespace=default \
-        --export > 01-source.yaml
-```
+- It's easy to locate things and know what and where is being deployed
+- If you need support from us, our engineers will easily understand the structure
 
-This command creates the Custom Resource for you and exports it to `01-source.yaml`. You can also use the `01-source.yaml` file from the example repository - they are identical. All resources generated in this document are stored there as well.
+The recommended structure (simplified) is:
 
-Time to apply the generated YAML:
-
-```nohighlight
-kubectl create -f 01-source.yaml
+```text
+bases
+├── clusters
+└── ...
+management-clusters
+└── MC_NAME
+    ├── secrets
+    └── organizations
+        └── ORG_NAME
+            └── workload-clusters
+                ├── WC_NAME1
+                │   ├── mapi                installed from the MC
+                │   │    ├── apps
+                │   │    │    └── APP1
+                │   │    └── cluster
+                │   └── [out-of-band]       installed directly in the WC
+                └── WC_NAME2                     
 ```
 
-You can see if the change was applied using `kubectl`:
+We provide a template repository where this structure is elaborated and explained in detail in [giantswarm/gitops-template](https://github.com/giantswarm/gitops-template).
+It's not necessary to clone that repo to follow this guide, but it can provide a good reference for every possible usecase.
+We recommend to copy, at least, the bases folder from that repository to your own repo because cluster creation uses them as templates as we will see later in this document.
 
```

Comment:
```suggestion
We provide a template repository where this structure is elaborated and explained in detail in [giantswarm/gitops-template](https://github.com/giantswarm/gitops-template). It is not necessary to clone the repository to follow this guide, but it can provide a good reference for every possible use case. We recommend copying, at least, the bases folder from that repository to your own repository because cluster creation uses them as templates as we will see later in this document.

```